build.sbt

```diff
@@ -20,10 +20,10 @@ ThisBuild / githubWorkflowBuildPreamble += {
   )
 }
 
-val scala2_12 = "2.12.18"
-val scala2_13 = "2.13.12"
-val scala3 = "3.3.1"
-val scalaNativeVersion = "0.4.14"
+val scala2_12 = "2.12.19"
+val scala2_13 = "2.13.13"
+val scala3 = "3.3.3"
+val scalaNativeVersion = "0.5.1"
```

Comment:
I don't want to bump this one to 0.5.x yet since that will force projects to upgrade. Will the changes you've made here still work with 0.4.x? At a glance I think so.

Reply:
Sure, sounds good!